repos/system_upgrade/common/actors/repositoriesmapping/libraries/repositoriesmapping.py

```diff
@@ -28,46 +139,36 @@ def scan_repositories(read_repofile_func=_read_repofile):
 
     See the description of the actor for more details.
     """
-    _exp_src_prod_type = config.get_product_type('source')
-    _exp_dst_prod_type = config.get_product_type('target')
-
-    repositories = []
-    line_num = 0
-    for line in read_repofile_func(REPOMAP_FILE)[1:]:
-        line_num += 1
-
-        # skip empty lines and comments
-        if not line or line.startswith('#'):
-            continue
-
-        try:
-            (from_repoid, to_repoid, to_pes_repo,
-             from_minor_version, to_minor_version, arch,
-             repo_type, src_prod_type, dst_prod_type) = line.split(',')
-
-            # filter out records irrelevant for this run
-            if (arch != api.current_actor().configuration.architecture
-                    or _exp_src_prod_type != src_prod_type
-                    or _exp_dst_prod_type != dst_prod_type):
-                continue
-
-            repositories.append(
-                RepositoryMap(
-                    from_repoid=from_repoid,
-                    to_repoid=to_repoid,
-                    to_pes_repo=to_pes_repo,
-                    from_minor_version=from_minor_version,
-                    to_minor_version=to_minor_version,
-                    arch=arch,
-                    repo_type=repo_type,
-                )
-            )
-        except (ModelViolationError, ValueError) as err:
-            _inhibit_upgrade('The repository mapping file is invalid, offending line number: {} ({}).'
-                             ' It is possible the file is out of date.'
-                             .format(line_num, err))
+    # NOTE: orig filter: skip different archs and product types
+    # NOTE: product type is changed, now it's eus,e4s,aus,tus,ga,beta
+    # NOTE: what about filter out all src repos that are not present on the sys?
+    # TODO: what to do when the repomap.csv is present? Inhibit, warn, report?
+    # # - as users could expect the old repomap is still used and could miss
+    # #   that the new repomap has been downloaded from the server (in example).
+    # TODO: deprecate the product type and introduce the "channels"
+    # NOTE: ... filter out everything what unrelated. Even better, why not to
+    # decide what is the expected mapping in this actor? So other actors alreay
+    # pick what is expected (including the setuptargetrepos actor).
+    # (sync with @shaded-enmity)
```

Comment:
We should address these comments - looks like they were highly relevant during the early stages of development, but that might have changed over time.